test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -328,5 +332,71 @@ public async Task RebuildIntellisenseCacheClearsScriptParseInfoCorrectly()
                 testDb.Cleanup();
             }
         }
+
+        [Fact]
+        // This test validates switching off editor intellisesnse for now. 
+        // Will change to better handling once we have specific SQLCMD intellisense in Language Service
+        public async Task HandleReuestTochangeToSQLCMDFile()
```

Comment:
>HandleReuestTochangeToSQLCMDFile [](start = 26, length = 32)

HandleRequestToChangeToSqlcmdFile

Reply:
A code spell checker extension in vscode had helped a lot with spelling issues. They are not perfect but they have helped me elminate 90% of my silly typos.